### PR TITLE
Use signat in check script

### PR DESCRIPTION
**Pull request by daradib**
Also fix indentation and move warning checks together.

### CODE REVIEW

File: acct/check [hunk 1]
```diff
@@ -52,25 +52,25 @@ else
...
+shell=$(ldapsearch -x -LLL uid=$user loginshell | grep ^loginShell | cut -d' ' -f2)
+if [ $shell = /opt/share/utils/bin/sorried ]; then
+  echo "WARNING: Account is disabled" >&2
+fi
+
 # check if Kerberos principal exists
 kerberos=$(echo TESTIFEXIST | kinit --password-file=STDIN $user 2>&1)
 if [ -n "$kerberos" ]; then
-  if [ -n "$(echo "$kerberos" | grep unknown$)" ]; then
+  if echo "$kerberos" | grep -q 'unknown$'; then
```
Comment (chriskuehl) [inline acct/check]: can we just make this bash lol

Comment (daradib) [inline acct/check]: feature creep ;)

File: acct/check [hunk 2]
```diff
@@ -96,30 +96,24 @@ fi
...
-   echo "Member of group(s): $groups"
+  echo "Member of group(s): $groups"
 fi
 
-# check if CalLink OID number exists in LDAP
-callinkOid=$(ldapsearch -x -LLL uid=$user callinkOid | grep -i ^callinkOid | cut -d' ' -f2)
-if [ -n "$callinkOid" ]; then
-   echo "CalLink OID number: $callinkOid"
-   if [ "$callinkOid" != 0 ]; then
-       echo "https://ocf.io/callinkapi?status=&category=&type=&name=&OrganizationID=$callinkOid"
```
Comment (daradib) [inline acct/check]: The URL? We can add it back but I think it belongs in `signat` instead.

Comment (chriskuehl) [inline acct/check]: i don't understand why it was deleted?

**Comment by tahabi** [comment]
I did this in #45 and neglected to check that this PR actually does the same thing, sorry about that :(

**Pull request closed without merging**